website/content/api-docs/api-structure.mdx

```diff
@@ -40,9 +40,9 @@ $ curl \
     http://127.0.0.1:8500/v1/agent/members
 ```
 
-Previously this was provided via a `?token=` query parameter. This functionality
-exists on many endpoints for backwards compatibility, but its use is **highly
+Previously this was provided via a `?token=` query parameter but its use is **highly
 discouraged**, since it can show up in access logs as part of the URL.
+This functionality is deprecated and will be removed in future versions.
```

Comment:
Questions for author and reviewers:
- Should we issue a warning in the Consul agent logs if someone attempts to use this deprecated parameter, stating that it will be removed in a future version?
- Should we call out the deprecation in the version-specific upgrade docs for 1.15? https://developer.hashicorp.com/consul/docs/upgrading/upgrade-specific

I'm leaning towards "yes" to both questions.

Reply:
@jkirschner-hashicorp I believe @roncodingenthusiast 's plan is to create a separate PR for the code changes addressing bullets 1 and 3.

Great catch on bullet 2, I totally agree!